ROADMAP.md

```diff
@@ -1,3 +1,29 @@
 # wasmCloud Roadmap and Vision
 
 The wasmCloud roadmap vision is now outlined in our documentation site. Please visit the [roadmap page](https://wasmcloud.com/docs/roadmap) for more information. Additionally, active work in progress and the status of new features is tracked using a GitHub project called [wasmCloud Roadmap](https://github.com/orgs/wasmCloud/projects/7/views/3).
+
+## Roadmap Process
+
+wasmCloud roadmaps are planned quarterly (every three calendar months) in the first community meeting of the quarter. If there is a holiday conflict or a majority of maintainers and contributors will be absent from a community meeting, then the roadmap session can be postponed to the following Wednesday. The purpose of the planning session is to review the previous quarter's roadmap for successes and areas of improvement, as well as to plan initiatives for wasmCloud contributors in the following quarter. It is strongly recommended that a wasmCloud organization maintainer leads the roadmapping session as they are responsible for understanding all areas of the project.
+
+The process for wasmCloud quarterly planning is as follows, and can be done by any wasmCloud organization maintainer:
+1. Create a GitHub Discussion at https://github.com/wasmCloud/wasmCloud/discussions (e.g. [https://github.com/wasmCloud/wasmCloud/discussions/3813](https://github.com/wasmCloud/wasmCloud/discussions/3813)) for community comments, requests and discussion. This should be the primary feedback mechanism for bringing in work for the next quarter. Optionally, if there is a high level goal of a roadmap (e.g. revamp documentation, add support for a new language) communicate that goal in the GitHub discussion.
+1. Schedule the community meeting and set the primary agenda item to be `DISCUSSION: wasmCloud QX Roadmap` (e.g. [https://wasmcloud.com/community/2024/10/02/community-meeting](https://wasmcloud.com/community/2024/10/02/community-meeting))
+1. For each major repository in wasmCloud, create an issue milestone for the applicable quarter e.g. `wasmCloud Q424`. Name the milestones the same in each repository for assistance in roadmap creation.
```

Comment:
For scalability purposes, would it just be easier to centralize it all in the GH project and not worry about milestones?

Reply:
Yes, I have a good idea here

Reply:
Made the change in https://github.com/orgs/wasmCloud/projects/7/views/12, now there's a `quarter` field on the milestone that we can use instead of the issue milestones